openshift-container-platform-4/policies/CA-Security_Assessment_and_Authorization/component.yaml

```diff
@@ -302,18 +302,16 @@
 - control_key: CA-9
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: not applicable
   narrative:
     - key: a
       text: |
-        Engineering progress can be tracked at:
-
-        https://issues.redhat.com/browse/CMP-256
+        CA-9 is an organizational control outside the scope
+        of OpenShift configuration.
     - key: b
       text: |
-        Engineering progress can be tracked at:
-
-        https://issues.redhat.com/browse/CMP-256
+        CA-9 is an organizational control outside the scope
+        of OpenShift configuration.
```

Comment:
The language of CA-9(b) at https://nvd.nist.gov/800-53/Rev4/control/ca-9:

``b. Documents, for each internal connection, the interface characteristics, security requirements, and the nature of the information communicated.``

We'll need to document what ports and protocols OpenShift uses. Is this something you could help with? Eg how the cluster elements connect to each other, over what protocols, what kind of data is shared, etc.